src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -4047,7 +4047,21 @@ LogicalResult ONNXRandomNormalOp::inferShapes(
 
 LogicalResult ONNXRandomNormalLikeOp::inferShapes(
     std::function<void(mlir::Region &)> doShapeInference) {
-  return emitError(NOT_IMPLEMENTED_MESSAGE);
+  if (!input().getType().isa<RankedTensorType>())
+    return success();
+  auto outputShape = input().getType().cast<RankedTensorType>().getShape();
+  auto elementTypeID = dtype().getValue();
+
+  RankedTensorType outputTensorType =
```

Comment:
The documentation (https://github.com/onnx/onnx/blob/main/docs/Operators.md#RandomNormalLike) says that the attribute `dtype` is optional and that when missing the output tensor element type is the same as the element type of the input tensor. Here it seems we would set it to `f32` in that situation.

How many values can `dtype` have? Is 2 the max value?